api/nakadi-event-bus-api.yaml

```diff
@@ -1132,6 +1161,13 @@ definitions:
           If set by the producer Event will be rejected.
         format: date-time
         example: '1996-12-19T16:39:57-08:00'
+      version:
```

Comment:
I don't know. As we don't expect this object to be submitted by the user - I think it doesn't make sense to mark fields as readOnly.